src/dictionary.ts

```diff
@@ -0,0 +1,115 @@
+import {isNullOrUndefined} from './sfdt/listHelpers';
+
```

Comment:
btw, If we want to make a true private members in javascript we should use closures.
> https://www.crockford.com/javascript/private.html